test/__init__.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-INDEX_NAME = "mediacloud_test"
+INDEX_NAME = "mc_search"
```

Comment:
The tests ran successfully on this pr! I think the fixtures use the INDEX_NAME defined here when they're created (which happens anew in every github action) - so it's ultimately all the same. I changed it to resemble the production index. name so that I could run a demonstration test on my end against the production database through a tunnel as I was having trouble getting a local ES running quickly.